packages/cucumber-example/test/support/setup.js

```diff
@@ -1,12 +1,14 @@
-import { setDefaultTimeout, AfterAll, BeforeAll } from 'cucumber';
-import { createSession, closeSession } from 'nightwatch-api';
+const { setDefaultTimeout, AfterAll, BeforeAll } = require('cucumber');
+const { createSession, closeSession, startWebDriver, stopWebDriver } = require('nightwatch-api');
 
 setDefaultTimeout(60000);
 
 BeforeAll(async () => {
-  await createSession({ env: 'default' });
+  await startWebDriver();
```

Comment:
You are not passing any environment as you want it to pickup default environment?

Reply:
Yes it will be "default" by default. But I have deleted this file.